packages/react-native-aztec/android/build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         robolectricVersion = '3.5.1'
         jUnitVersion = '4.12'
         jSoupVersion = '1.10.3'
-        wordpressUtilsVersion = '1.22'
+        wordpressUtilsVersion = '2.3.0'
```

Comment:
👋 @oguzkocer , looks like [the WPAndroid app is on 2.2.0](https://github.com/wordpress-mobile/WordPress-Android/blob/ecb4fa87f770557e7db03861854e2f7eaa8aba98/build.gradle#L3) still, do you think it might be an issue to have 2.3.0 brought in by the Gutenberg mobile lib?

Reply:
@hypest I'll open a PR to upgrade it to `2.3.0`, shouldn't be a problem. We can merge this PR in advance.

Reply:
Works for me! 👍